prettier/prettier.config.js

```diff
@@ -8,5 +8,5 @@ module.exports = {
   bracketSpacing: true,
   arrowParens: 'always',
   proseWrap: 'preserve',
-  htmlWhitespaceSensitivity: 'css',
-};
\ No newline at end of file
+  htmlWhitespaceSensitivity: 'ignore',
```

Comment:
Do we plan to change all our Covalent components for `preservewhitespaces : false` because of this change?

Reply:
Not quite. `preservewhitespaces` refers to preserving whitespace during build time. `htmlWhitespaceSensitivity` .

Reply:
I understand that. was asking the question based on this statement: `In angular components, [preservewhitespaces] defaults to `false`, so just default to false as well.` if we are planning that for our components too.

Reply:
`preserveWhitespaces` is used in many components, probably in some components that don't even require it. So we might be able to get rid of some usage. The only places that require it is when we use `<td-highlight/>`.  But even then, we might be better off using the `ngPreserveWhitespaces` directive or `<pre/>` tag instead.